oak_functions/client/src/main.rs

```diff
@@ -33,24 +34,37 @@ pub struct Opt {
     uri: String,
     #[structopt(long, help = "request payload")]
     request: String,
+    #[structopt(long, help = "expected response body")]
+    expected_response: Option<String>,
```

Comment:
Could you add a comment that it's needed for testing?

Reply:
Done.